test-framework/common/src/main/java/io/quarkus/test/common/DefaultDockerContainerLauncher.java

```diff
@@ -24,15 +21,17 @@
 import java.util.concurrent.TimeUnit;
 import java.util.function.Function;
 
-import org.apache.commons.io.input.TeeInputStream;
 import org.apache.commons.lang3.RandomStringUtils;
+import org.jboss.logging.Logger;
 
 import io.quarkus.runtime.util.ContainerRuntimeUtil;
 import io.quarkus.test.common.http.TestHTTPResourceManager;
 import io.smallrye.config.common.utils.StringUtil;
 
 public class DefaultDockerContainerLauncher implements DockerContainerArtifactLauncher {
 
+    private static final Logger log = Logger.getLogger(DefaultDockerContainerLauncher.class);
```

Comment:
Are you sure the logger works properly?

Reply:
Well...it logs:
```
[INFO] Running io.quarkus.awt.it.ImageDecodersIT
2023-02-28 12:36:39,352 INFO  [io.qua.tes.com.DefaultDockerContainerLauncher] (main) 
Executing "podman run --name quarkus-integration-test-srEHL -i --rm -p 8081:8081 -p 8444:8444 --net=quarkus-integration-test-qvkwH --env QUARKUS_HTTP_PORT=8081 --env QUARKUS_HTTP_SSL_PORT=8444 --env TEST_URL=http://localhost:8081 karm/quarkus-integration-test-awt:999-SNAPSHOT"
```

So I'd say it's O.K.